android/src/main/java/com/visioncameracodescanner/VisionCameraCodeScannerPackage.java

```diff
@@ -19,8 +19,8 @@ public class VisionCameraCodeScannerPackage implements ReactPackage {
   public VisionCameraCodeScannerPackage() {
```

Comment:
it appears that as per [this pr](https://github.com/mrousavy/react-native-vision-camera/pull/2039) on rnvc, the plugin should be initialized in a `static` block of the package.